core/src/main/java/com/chopshop166/chopshoplib/motors/PIDSparkMax.java

```diff
@@ -70,6 +70,15 @@ public SparkMaxPIDController getPidController() {
         return sparkPID;
     }
 
+    /**
+     * Add a validator to make sure that the current is below a provided limit.
+     * 
+     * @param limit The maximum current to allow.
+     */
+    public void validateCurrent(final double limit) {
```

Comment:
Shouldn't the validateCurrent method also be in SmartMotorController? This would make it accessible to any subsystems that only use the SmartMotorController class.

Reply:
I also feel like there should be a way to access the current through SmartMotorController to be able to display it on the dashboard to find an appropriate limit.

Reply:
That makes sense. With how modifiers currently are for motors, is there a way to easily use validators with modifiers? If not, it might be useful to add some sort of single limit to modifiers which then could be used with validators.

Reply:
Take a loot at c39c84c and see if that modifier will do what you want?

Reply:
Yeah that is what I was looking for.

Reply:
We can get the current for each channel on the PDP/PDH which we could probably use to implement this for the basic motor controllers. This could be an interesting exercise for someone with free time. But since we don't generally use these types of controllers its not worth prioritizing.